Services/Mail/classes/class.ilObjMailGUI.php

```diff
@@ -539,14 +539,19 @@ protected function getExternalSettingsForm()
 	 */
 	protected function populateExternalSettingsForm(ilPropertyFormGUI $form)
 	{
+		$subjectPrefix = $this->settings->get('mail_subject_prefix');
+		if (false === $subjectPrefix) {
```

Comment:
@legionth Do we still need this? For the `new installations should always be delivered with "[ILIAS]" in the mail subject` requirement we should create a database update step.

Furthemore we need an addition for the byline: `We recommend the usage of a prefix for supporting message filtering by users.`

Reply:
Because of the strict type check on a new system `$this->settings->get('mail_subject_prefix');` will return false, because this parameter is not set in the settings tables.
After the first save, we will always have a value stored in there.
Yea let's discuss this tomorrow :+1: 

Sure I will add a info! :+1:

Reply:
OK, thanks for clarification. In this case we should maybe also change `\ilMail::getSubjectPrefix`.